From: NIIBE Yutaka <gniibe@fsij.org>
Date: Tue, 7 Mar 2017 09:57:16 +0900
Subject: w32: Conditionalize ISO 2022 definitions.

* src/w32-iconv.c (iso2022_SI_seq, iso2022_SO_seq) [USE_MLANG_DLL]:
Only for USE_MLANG_DLL.

--

Signed-off-by: NIIBE Yutaka <gniibe@fsij.org>
(cherry picked from commit 1126c4c117a47c8ea8435ac11561d51b13f538dd)
---
 src/w32-iconv.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/w32-iconv.c b/src/w32-iconv.c
index c10bb32..888b722 100644
--- a/src/w32-iconv.c
+++ b/src/w32-iconv.c
@@ -1475,6 +1475,7 @@ utf32_wctomb(csconv_t *cv, ushort *wbuf, int wbufsize, uchar *buf, int bufsize)
 #define ISO2022_SI  0
 #define ISO2022_SO  1
 
+#if USE_MLANG_DLL
 /* shift in */
 static const char iso2022_SI_seq[] = "\x0F";
 /* shift out */
@@ -1487,6 +1488,7 @@ struct iso2022_esc_t {
     int len;
     int cs;
 };
+#endif
 
 #define ISO2022JP_CS_ASCII            0
 #define ISO2022JP_CS_JISX0201_ROMAN   1
